[FORMATS-146] Add variant quality field.

fnothaft opened the pull request:
Resolves #146.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/bdg-formats-prb/167/
Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/bdg-formats-prb/168/
Test PASSed.

heuermh commented on src/main/resources/avro/bdg.avdl:
```diff
@@ -765,6 +765,12 @@ record Variant {
   union { null, string } alternateAllele = null;
 
   /**
+   The Phred scaled probability that this allele is variant. VCF column 6.
```
minor nit, `VCF column 6.` &rarr; `VCF column 6 "QUAL".`

heuermh commented:
Then about releases, this will be required for ADAM 0.23.0, so we cut a 0.11.2 for bdg-formats with only this PR in it?

fnothaft commented:
Addressed review comments.

> Then about releases, this will be required for ADAM 0.23.0, so we cut a 0.11.2 for bdg-formats with only this PR in it?

Yup!

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/bdg-formats-prb/169/
Test PASSed.

heuermh commented:
Thank you, @fnothaft!
